README.md

```diff
@@ -79,3 +79,4 @@ Please check [CONTRIBUTING.md](https://github.com/espoo-dev/espoo-dev/blob/main/
 - Answers reports
 - Public surveys
 - Surveys protected by password
+
```

Comment:
@philipeleandro remove this extra empty line

Reply:
@edimossilva It was removed.